src/app/AppLayout/SslErrorModal.tsx

```diff
@@ -38,7 +38,7 @@ export const SslErrorModal: React.FC<SslErrorModalProps> = ({ visible, onDismiss
       variant={ModalVariant.medium}
       showClose={true}
       onClose={onDismiss}
-      title="SSL Error"
+      title="SSL error"
```

Comment:
Maybe now or maybe in a follow-up PR, but wherever we mention `SSL` it should probably be `SSL/TLS`

Reply:
I think I should be able to do it in this PR.